test/Chainweb/Test/Pact/Utils.hs

```diff
@@ -47,6 +52,40 @@ module Chainweb.Test.Pact.Utils
 , withPayloadDb
 , withBlockHeaderDb
 , withTemporaryDir
+-- * 'PactValue' helpers
```

Comment:
It's not worth holding up this PR for, but this stuff looks fairly similar to some of the transaction building stuff in the tx generator.  I guess we recently pulled that code out into a separate repo and maybe these are chainweb-specific enough that it doesn't make sense, but it might be worth keeping in mind for consolidation at some point down the road if possible.

Reply:
Unless I'm missing something, TXG uses Pact's `ApiReq` which while it is a builder, it's optimized for YAMLs, so there's a million `Maybe`s. The approach here is to provide a principled un-signed builder.

What are you referring to in TXG? https://github.com/kadena-io/txg

Reply:
Nothing specific.  It was just a notion of vaguely similar things floating around, and I wanted to mention it if for nothing more than to increase awareness...especially since it's in a different repo now.